main/getting-started/start-a-project.md

```diff
@@ -22,24 +22,23 @@ Also, for what needs to be done after you modify a project's code, see
   </th>
   <tr>
     <td><center>1</center></td>
-    <td>Go to or open a shell and <code>cd ~/agoric-sdk</code></td>
-    <td>Go to the directory where you copied the latest Agoric SDK
-	    from the Agoric GitHub repository in Step 4 of <a href="https://agoric.com/documentation/getting-started/before-using-agoric/">Before Using Agoric</a>.</td>
+    <td>Go to or open a shell and <code>cd &lt;directory-where-you-want-to-install-Dapp-code&gt;</code></td>
+    <td>When you initialize your project/Dapp in the next step, its initial code is copied into your current directory (permissions allowing).</td>
   </tr>
   <tr>
     <td><center>2</center></td>
     <td>Run <code>agoric init demo</code>
     <td>Initializes a <i>Dapp</i> (<i>Distributed application</i>)
 	project.
 	<br><br>
-    <code>init</code> creates a folder with the specified name
+    <code>init</code> creates a sub-directory with the specified name
     (<code>demo</code> in this case) in your current directory and copies an existing project's files
     into it. You can give the project any name you like. This
     documentation assumes you used <code>demo</code>. 
     <br><br>
     There are optional arguments that let you specify which project
     you want copied into the specified directory. By default, their values are set to
-    use a Dapp we wrote to introduce you to Agoric smart
+    use the Encouragement Dapp we wrote to introduce you to Agoric smart
```

Comment:
"we wrote to provide a simple skeleton for a smart contract."  this is because encouragement doesn't really introduce much about smart contracts, but it does show a lot about how to set up the pieces.